diff --git a/src/Illuminate/Queue/RedisQueue.php b/src/Illuminate/Queue/RedisQueue.php
index a4d9499b6..7bda32fb8 100644
--- a/src/Illuminate/Queue/RedisQueue.php
+++ b/src/Illuminate/Queue/RedisQueue.php
@@ -229,7 +229,7 @@ class RedisQueue extends Queue implements QueueContract
     {
         $rawBody = $this->getConnection()->blpop($queue, $this->blockFor);

-        if (! is_null($rawBody)) {
+        if (! empty($rawBody)) {
             $payload = json_decode($rawBody[1], true);

             $payload['attempts']++;
